src/khoj/interface/web/login.html

```diff
@@ -144,10 +135,11 @@ <h1 class="login-modal-title">Become superhuman with your personal knowledge bas
             display: block;
         }
 
-        h1.login-modal-title {
+        .login-modal-title {
```

Comment:
It's better to be specific with styling options so we can make the intent clear and prevent unintended side effects. Is there a reason why you're removing the `h1` here?